packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -564,7 +564,7 @@ class TextPainter {
     _paragraph.layout(ui.ParagraphConstraints(width: maxWidth));
     if (minWidth != maxWidth) {
```

Comment:
Just want to make sure I'm understanding this part correctly: it shrink-wraps the `Paragraph` to `[minWidth, maxWidth]` if it occupies only one line?

Reply:
I think it does so even for multiline paragraphs if I'm reading the code correctly.